src/js/modal-dialog.js

```diff
@@ -119,14 +119,18 @@ class ModalDialog extends Component {
    * Handles `keydown` events on the document, looking for ESC, which closes
    * the modal.
    *
-   * @param {EventTarget~Event} e
+   * @param {EventTarget~Event} event
    *        The keypress that triggered this event.
    *
    * @listens keydown
    */
-  handleKeyPress(e) {
-    if (e.which === ESC && this.closeable()) {
+  handleKeyPress(event) {
+    if (keycode.isEventKey(event, 'Escape') && this.closeable()) {
       this.close();
+    } else {
+
+      // Pass keypress handling up for unsupported keys
```

Comment:
Do we want to pass keyboard events to the player while the modal is over the player?

Reply:
Good question! If not, then we need to change the CloseButton as well, because it extends Button which *does* pass keyboard events up to the player.

But yes, now I think about it, the whole point of a "modal" is that it makes the thing behind it (usually the whole window, but in this case just the player, *inert*. I'll change that, and the CloseButton.